### ISSUE TITLE
Unknown option: direct.browserPolyfill

### ISSUE THREAD

**Issue by joshuairl** [babel/broccoli-babel-transpiler#41]
I'm getting an error when using browserPolyfill option in my Brocfile.js.
Is this still supported?

```
ReferenceError: [BABEL] index.js: Unknown option: direct.browserPolyfill
    at Logger.error (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:124:29)
    at OptionManager.init (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/options/option-manager.js:216:10)
    at File.initOptions (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/index.js:146:75)
    at new File (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/file/index.js:133:22)
    at Pipeline.transform (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Babel.transform (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/index.js:61:21)
    at Babel.processString (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/index.js:73:25)
    at Babel.processFile (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/cauliflower-filter/filter.js:166:31)
    at asyncProcessFile (/Users/Joshua/Projects/kurento-room-js/node_modules/broccoli-babel-transpiler/node_modules/cauliflower-filter/filter.js:101:21)
```

**Comment by stefanpenner** [comment]
> Is this still supported?

yes

> I'm getting an error when using browserPolyfill option in my Brocfile.js.

can you show how you are doing this?

**Comment by stefanpenner** [comment]
reproduced it

**Comment by stefanpenner** [comment]
cc @givanse

**Comment by stefanpenner** [comment]
ah this isn't yet released, but it needs some fixes. Will work on it tonight

**Comment by stefanpenner** [comment]
published as 5.3.0

**Comment by givanse** [comment]
@stefanpenner with "some fixes" do you mean adding a test for `browserPolyfill`? or something else? (module metadata tests are failing)
